school_project/src/main/java/Main.java

```diff
@@ -0,0 +1,7 @@
+public class Main {
```

Comment:
Why u need Main ?

Reply:
I thought it is needed to be able to run other methods. But I see that tests could be run without Main class. Removed Main,